### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by datakurre**
This is mainly to rebuild installers at CI

### CODE REVIEW

**Comment by datakurre** [comment]
And all builds are broken again... initially it looks like mostly Chrome version mismatch on OSX and Selenium timing issues (Azure is slower than previously) on all systems.

**Comment by datakurre** [comment]
Stable Chrome on MacOS is now 74 and matching chromedriver is missing Python 3.6 build for MacOS https://anaconda.org/conda-forge/python-chromedriver-binary/files :see_no_evil:

File: recipes/robotlab/meta.yaml [hunk 1]
```diff
@@ -69,6 +69,7 @@ test:
     - robotlab --version
     - robotlab-extension list
     - robotlab-examples
+    - dir  # [win]
```
Comment (bollwyvl) [inline recipes/robotlab/meta.yaml:72]: Never seen `dir` before..

File: tests/acceptance/RobotLab/02_Robot_Notebook.robot [hunk 2]
```diff
@@ -34,8 +34,9 @@ Can RobotLab make a Robot Notebook?
 The Robot Popup Should Contain
     [Arguments]    ${prefix}    ${document}    ${msg}
     [Documentation]    With an open Robot Notebook, take a look at the log or report
+    Sleep    2s
     Click Link    ${document}
-    Sleep    1s
+    Sleep    2s
```
Comment (bollwyvl) [inline tests/acceptance/RobotLab/02_Robot_Notebook.robot:39]: Should we globalize some wait numbers?

**Comment by bollwyvl** [comment]
> @datakurre Stable Chrome on MacOS is now 74 and matching chromedriver is missing Python 3.6 build for MacOS :see_no_evil:

https://anaconda.org/conda-forge/python-chromedriver-binary/files?version=74.0.3729.6.0 :monkey_face:

File: recipes/robotlab/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/robotlab/meta.yaml:72]: o, yar, it's not a package. Never mind!

**Comment by datakurre** [comment]
@bollwyvl Thanks!

None of my changes here should be taken seriously....

Windows build stopped on command "robotlab-tutorial" and I had now idea why.

And those "sleeps" are not a real solution neither...

**Pull request closed without merging**